weave-js/src/components/PagePanelComponents/Home/Browse3/compare/CompareGridCellValue.tsx

```diff
@@ -71,12 +71,12 @@ export const CompareGridCellValue = ({
       </div>
     );
   }
-
+  const leafPath = path.toString().split('.').slice(-1)[0] ?? '';
```

Comment:
I think there's a cleaner way to write this given the methods on ObjectPath, maybe `path.tail()` or `path.toStringArray().pop()`